python/tests/res/enkf/test_programmatic_res_config.py

```diff
@@ -336,8 +336,8 @@ def test_minimum_config(self):
             self.assertEqual(loaded_res_config.model_config.getEnspath(),
                              prog_res_config.model_config.getEnspath())
 
-            self.assertEqual(loaded_res_config.site_config.queue_config.job_script,
-                             prog_res_config.site_config.queue_config.job_script)
+            self.assertEqual(loaded_res_config.queue_config.job_script,
```

Comment:
Is this not superseeded by the equality of the two `queue_configs` below?

Reply:
equality of queue configs in test_large_config